doc/classes/Image.xml

```diff
@@ -291,25 +311,28 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Load an [code]Image[/code].
+				Loads an image from file [code]path[/code].
 			</description>
 		</method>
 		<method name="lock">
 			<return type="void">
 			</return>
 			<description>
+				Locks the data and prevents writing access.
```

Comment:
- "writing access" -> changes

Reply:
Lock also prevents reading, like `get_pixel`, so an example might be needed in `get_pixel` and `set_pixel`

something like
```
var img = Image.new()
img.unlock()
var c = img.get_pixel(x, y)
img.lock()
```